src/qt/transactionrecord.cpp

```diff
@@ -42,17 +42,17 @@ QList<TransactionRecord> TransactionRecord::decomposeTransaction(const CWallet *
     uint256 hash = wtx.GetHash();
     std::map<std::string, std::string> mapValue = wtx.mapValue;
 
-    bool isAllSigmaSpendFromMe = wtx.IsZerocoinSpendV3();
+    bool isAllSigmaSpendFromMe;
 
     for (const auto& vin : wtx.vin) {
-        isAllSigmaSpendFromMe = wallet->IsMine(vin) & ISMINE_SPENDABLE;
+        isAllSigmaSpendFromMe = (wallet->IsMine(vin) & ISMINE_SPENDABLE) && vin.IsZerocoinSpendV3();
```

Comment:
Maybe set isAllSigmaSpendFromMe  = true, then change this line to isAllSigmaSpendFromMe &&=